lib/helpers.js

```diff
@@ -59,10 +59,10 @@ export async function createCapabilities({profileId, request}) {
     const keyPair = await Ed25519VerificationKey2020.from(keyDescription);
     const fingerprint = keyPair.fingerprint();
 
-    if(profileId.includes('did:key:')) {
+    if(profileId.startsWith('did:key:')) {
       invocationTarget.publicAlias =
         `did:key:${fingerprint}#${fingerprint}`;
-    } else if(profileId.includes('did:v1')) {
+    } else if(profileId.startsWith('did:v1')) {
```

Comment:
Should this be `did:v1:` or was it trying to check for other `v1xxx` strings?

Reply:
It should be `did:v1:`.